src/lib/FullCalendar.svelte

```diff
@@ -123,12 +123,9 @@
 >
 	<article>
 		<h2>Add to Your Calendar</h2>
-		<p>
-			Copy the iCal url into your calendar app to automatically add Kendo Club events to your
```

Comment:
We might still want to let them know that they need to copy it.  What about "Copy this iCal url into your calendar app to subscribe to the kendo at umich calendar" I don't know.

Reply:
That's really good, yeah!